modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/batch/engine/ColumnDescriptorProviderImpl.java

```diff
@@ -83,6 +93,80 @@ public ColumnDescriptor[] getColumnDescriptors(
 		};
 	}
 
+	private ColumnDescriptor[] _getAttachmentColumnDescriptors(
+		String fieldName, int index,
+		ObjectValuePair<Field, Method> propertiesObjectValuePair) {
+
+		ColumnDescriptor[] attachmentColumnDescriptors =
+			new ColumnDescriptor[4];
+
+		attachmentColumnDescriptors[0] = ColumnDescriptor.from(
```

Comment:
ColumnDescriptor object contains the relevant information for a related column in batch engine export csv file. The UnsafeFunction of ColumnDescriptor is used to extract field value from a batch item to be written to the csv column. This means for each batch item that is exported, the UnsafeFunction will be called, e.g. if we have 1mil items, the unsafe function will be called 1 mil times.

For this reason, I'm trying to avoid unnecessary if conditions as much as possible. I understand from the perspective of duplicated code/DRY principle this might look wrong, but here performance has higher priority and I would leave  the lambdas separated instead of merging them at the cost of adding one if condition.